ui/src/app/views/breadcrumb/component.ts

```diff
@@ -78,7 +78,10 @@ export class BreadcrumbComponent implements OnInit, OnDestroy {
       if (routeURL.length > 0) {
         url += `/${routeURL}`;
       }
-      const label = child.snapshot.data[ROUTE_DATA_BREADCRUMB];
+      let label = child.snapshot.data[ROUTE_DATA_BREADCRUMB];
+      if (label === 'Param: Workspace Name') {
+        label = child.snapshot.data['workspace'].name;
```

Comment:
Oooh ... this should work now because we've pushed the workspace all the way down through to the lowest child with the new route configuration.